internal/network/machine_network_cidr.go

```diff
@@ -417,6 +418,64 @@ func GetInventoryNetworksByFamily(hosts []*models.Host, log logrus.FieldLogger)
 	return ret, nil
 }
 
+func GetDefaultRouteNetworkByFamily(h *models.Host, networks map[AddressFamily][]string, log logrus.FieldLogger) (map[AddressFamily]string, error) {
+	var err error
+	ret := make(map[AddressFamily]string)
+
+	//start with dummy route
+	defaultRoutev4 := &models.Route{Metric: math.MaxInt32}
+	defaultRoutev6 := &models.Route{Metric: math.MaxInt32}
+
+	if h.Inventory != "" {
+		var inventory models.Inventory
+		err = json.Unmarshal([]byte(h.Inventory), &inventory)
+		if err != nil {
+			log.WithError(err).Warnf("Unmarshal inventory %s", h.Inventory)
+			return ret, err
+		}
+		//find the minimal default route
+		var isDefault, found bool
+		for _, route := range inventory.Routes {
+			if isDefault, err = IsDefaultRoute(route); err != nil {
+				log.WithError(err).Errorf("fail to find default route")
+				return ret, err
+			}
+			if isDefault {
+				if IsIPv4Addr(route.Gateway) && route.Metric < defaultRoutev4.Metric {
+					log.Info("==> found ipv4 route")
```

Comment:
Is this staying ?

Reply:
removed and replaced by a log at the end of the function